catroid/src/main/java/org/catrobat/catroid/ui/dialogs/FormulaEditorComputeDialog.java

```diff
@@ -144,10 +143,8 @@ public void run() {
 
 				ViewGroup.LayoutParams params = computeTextView.getLayoutParams();
 				int heightPixels = computeTextView.getLineCount() * computeTextView.getLineHeight();
-				int height = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, heightPixels / 2, context
-						.getResources().getDisplayMetrics());
 				params.width = ViewGroup.LayoutParams.FILL_PARENT;
-				params.height = height;
+				params.height = heightPixels + (int) (heightPixels * 0.5); //set line height + 50% margin height
```

Comment:
please remove the comment, and instead either go with:
float marginFactor = 1.5;
params.height = (int) (heightPixels * marginFactor );
or:
int heightMargin = heightPixels * 0.5;
params.height = heightPixels + heightMargin;
so it becomes clear without the comment whats going on here

Reply:
Robin: "Let's go! We can finish this task easy!" 
Batman: "Easily." 
Robin: "Easily." 
Batman: "Good grammar is essential, Robin." 
Robin: "Thank you." 
Batman: "You're welcome."